controllers/storagecluster/storagecluster_controller.go

```diff
@@ -7,7 +7,12 @@ import (
 	"reflect"
 
 	"github.com/go-logr/logr"
+	"github.com/google/go-cmp/cmp"
+	"github.com/google/go-cmp/cmp/cmpopts"
```

Comment:
was tidy already run? I don't see these mod jumping to `require` section w/o `indirect` flag.

Reply:
Just did that